prov/util/src/util_mem_monitor.c

```diff
@@ -341,3 +362,74 @@ void ofi_uffd_cleanup(void)
 }
 
 #endif /* HAVE_UFFD_UNMAP */
+
+/*Lets define what we define in order to launch configure.ac*/
+//#if
+
+static int ofi_patcher_register(const void *addr, size_t len, size_t page_size)
+{
+
+	int ret;
+	/*TODO*/
+	return 0;
+}
+
+static int ofi_patcher_subscribe(struct ofi_mem_monitor *monitor,
+			         const void *addr, size_t len)
+{
+	int i;
+
+	assert(monitor == &patcher.monitor);
+	for (i = 0; i < num_page_sizes; i++) {
+		if (!ofi_patcher_register(addr, len, page_sizes[i]))
+			return 0;
+	}
+	return -FI_EFAULT;
+}
+
+static int ofi_patcher_unregister(const void *addr, size_t len, size_t page_size)
+{
+	int ret;
+	/*TODO*/
+	return 0;
+}
+
+static void ofi_patcher_unsubscribe(struct ofi_mem_monitor *monitor,
+				    const void *addr, size_t len)
+{
+	int i;
+
+	assert(monitor == &patcher.monitor);
+	for (i = 0; i < num_page_sizes; i++) {
+		if (!ofi_patcher_unregister(addr, len, page_sizes[i]))
+			break;
+	}
+}
```

Comment:
Because patcher intercepts all calls that may modify the virtual address to physical address mappings, the subscribe/unsubscribe calls aren't needed.  They can be no-ops.  Subscribe can just return 0.

Reply:
Done it for now.